pkg/coscheduling/coscheduling.go

```diff
@@ -55,15 +55,31 @@ var _ framework.PostBindPlugin = &Coscheduling{}
 
 const (
 	// Name is the name of the plugin used in Registry and configurations.
-	Name = "Coscheduling"
+	Name                                       = "Coscheduling"
+	defaultPermitWaitingTimeSeconds      int64 = 60
+	defaultDeniedPGExpirationTimeSeconds int64 = 20
```

Comment:
We can remove this, as `pkg/apis/config/v1beta1/defaults.go` has already done the defalting logic.

Reply:
We'll then need to import the package`config/v1beta1`.  In that case, we should probably just use `SetDefaultsCoschedulingArgs` instead.

Reply:
> We can remove this, as `pkg/apis/config/v1beta1/defaults.go` has already done the defalting logic.

We'll then need to import the package`config/v1beta1`.  In that case, we should probably just use `SetDefaultsCoschedulingArgs` instead.

Reply:
I'm not sure I'm following. In terms of args, aren't we already using `v1beta1`, and `SetDefaultsCoschedulingArgs` is called implicitly?

https://github.com/kubernetes-sigs/scheduler-plugins/blob/011599fb15f2d57dbec7afa086814602d7f24664/pkg/apis/config/v1beta1/register.go#L29